src/main/java/com/raoulvdberge/refinedstorage/gui/grid/sorting/GridSortingInventoryTweaks.java

```diff
@@ -0,0 +1,33 @@
+package com.raoulvdberge.refinedstorage.gui.grid.sorting;
+
+import com.raoulvdberge.refinedstorage.apiimpl.network.node.NetworkNodeGrid;
+import com.raoulvdberge.refinedstorage.gui.grid.stack.GridStackItem;
+import com.raoulvdberge.refinedstorage.gui.grid.stack.IGridStack;
+import invtweaks.api.InvTweaksAPI;
+import net.minecraftforge.fml.common.Loader;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+
+public class GridSortingInventoryTweaks extends GridSorting {
+    private InvTweaksAPI api = null;
+
+    public GridSortingInventoryTweaks() {
+        try {
+            api = (InvTweaksAPI) Class.forName("invtweaks.forge.InvTweaksMod", true, Loader.instance().getModClassLoader()).getField("instance").get(null);
+        } catch (Exception ex) {
+            LogManager.getLogger(getClass()).log(Level.ERROR, "Error with InventoryTweak sorting", ex);
```

Comment:
Other than it being bad form generally to catch Java exceptions and do nothing with them, I agree that with the  few changes, the logging I had is rendered a bit superfluous since it no longer encapsulates the entire sorting calls, so sure I'll remove it.